apps/i18n/common/en_us.json

```diff
@@ -484,6 +484,13 @@
   "coteacherAdd": "Add Co-Teachers",
   "coteacherAddTooltip": "Co-teachers have the same access as you in managing this section and viewing student work. Co-teachers will see the invitation on their teacher dashboard.",
   "coteacherAddInfo": "Add co-teachers by entering the email address associated with their Code.org account in the field below. Each section can have up to five co-teachers.",
+  "coteacherAddNoEmail": "Please enter an email address.",
+  "coteacherAddInvalidEmail": "{email} is not a valid email address.",
+  "coteacherAddAlreadyExists": "{email} is already a co-teacher for this section.",
```

Comment:
Just checking, I know this is more of a design question, but if I were a teacher who got this error, my next question question would be "now what?" - do we have a tool tip or something that directs a teacher to do something if they get this error?  Something like "The co-teacher with this email address has already been notified and can find their invitations at X"?

Reply:
Below the input there will be a table including the names/emails and statuses of all the coteacher invitations. So the teacher would be able to look down at the table and see whether that teacher has already accepted, has declined or needs to accept.